src/main/java/org/vafer/jdependency/Clazzpath.java

```diff
@@ -162,12 +153,12 @@ public void remove() {
                     };
                 }
             }, pId, false);
-        } finally {
```

Comment:
It seems like the input stream does not closed anymore?

Reply:
It takes advantage of Java7 `try-resource`:

```java
try(final JarInputStream inputStream = new JarInputStream(pInputStream)) {
```